src/components/ErrorWrapper.jsx

```diff
@@ -5,7 +5,7 @@ export default function ErrorWrapper({ header, error }) {
   return (
     <Message negative>
       <Message.Header>{header}</Message.Header>
-      {error && <p>{error.info?.message || error.message || error}</p>}
+      {error && error !== '' && <p>{error}</p>}
```

Comment:
when error === '', it is considered as true

ergo, we can just write `error && <p>{error}</p>`